integration_tests/src/main/python/collection_ops_test.py

```diff
@@ -27,6 +27,8 @@
 non_nested_array_gens = [ArrayGen(sub_gen, nullable=nullable)
                          for nullable in [True, False] for sub_gen in all_gen + [null_gen]]
 
+non_null_nested_array_gens = [g for g in non_nested_array_gens if not g.nullable]
```

Comment:
why non-null? If we cannot support null ordering properly we either have to fall back to the CPU when we see it or we have to fix it in cudf.  Becuase we don't really have a customer asking for this right now I would rather see us fall back to the CPU.

Reply:
Sorry, it shouldn't be non-null specific at here. I removed the non-null constriction on test data.